README.md

```diff
@@ -1,13 +1,13 @@
 # prebuild-install [![Build Status](https://travis-ci.org/mafintosh/prebuild-install.svg?branch=master)](https://travis-ci.org/mafintosh/prebuild-install)
 
-A command line tool for easily install prebuilds for multiple version of node/iojs on a specific platform.
+Command line tool to easily install prebuilds for multiple version of node/iojs
```

Comment:
I try to stick to the standard limit of 80 columns everywhere (also my thesis I did it in 80 columns!), so it's easier to read and scroll when reading the source code.